DCA/colors.py

```diff
@@ -0,0 +1,40 @@
+from PIL import Image
+import os
+import argparse
+from pathlib import  Path
```

Comment:
To avoid manually aligning all the spaces, it is better to use a linter. There are two spaces here